webpack/assets/javascripts/react_app/redux/actions/common/index.js

```diff
@@ -11,5 +11,5 @@ export const ajaxRequestAction = ({
   dispatch({ type: requestAction, payload: item });
   return API.get(url)
     .then(({ data }) => dispatch({ type: successAction, payload: { ...item, ...data } }))
-    .catch(error => dispatch({ type: failedAction, payload: { error, item } }));
+    .catch(error => dispatch({ type: failedAction, payload: { error: error.message, item } }));
```

Comment:
I have one concern about storing only the error message:
the message property  is optional, and could be blank, plus if we'd like to distinct between error types we couldn't do it only with the error's message.